src/clj/rems/api/catalogue_items.clj

```diff
@@ -48,7 +48,7 @@
 
 (s/defschema ChangeFormResponse
   {:success s/Bool
-   :new-catalogue-item-id s/Int})
+   :catalogue-item-id s/Int})
```

Comment:
This breaking API change should be mentioned in the changelog.

Reply:
It's not a breaking API change when this is the PR where the API is introduced in.

Reply:
Ok.